tests/t_json

```diff
@@ -12,14 +12,14 @@ test_json() {
     PARAMS="$1"
     EXPECTED=$(echo "$2" | jq 'sort')
     RESULT=$($SDCV $PARAMS | jq 'sort')
-    if [ "$EXPECTED" != "$RESULT"]; then
+    if [ "$EXPECTED" != "$RESULT" ]; then
         echo "expected $EXPECTED but got $RESULT"
         exit 1
     fi
 }
 
-test_json "-x -j -l -n --data-dir \"$TEST_DIR\"" "[{\"name\": \"Test synonyms\", \"wordcount\": \"1\"},{\"name\": \"Sample 1 test dictionary\", \"wordcount\": \"1\"},{\"name\": \"test_dict\", \"wordcount\": \"1\"}]"
-test_json "-x -j -n --data-dir \"$TEST_DIR\" foo" "[{\"dict\": \"Test synonyms\",\"word\":\"test\",\"definition\":\"\nresult of test\"}]"
-test_json "-x -j -n --data-dir \"$TEST_DIR\" foobarbaaz" "[]"
+test_json "-x -j -l -n --data-dir $TEST_DIR" "[{\"name\": \"Test synonyms\", \"wordcount\": \"2\"},{\"name\": \"Sample 1 test dictionary\", \"wordcount\": \"1\"},{\"name\": \"test_dict\", \"wordcount\": \"1\"}]"
+test_json "-x -j -n --data-dir $TEST_DIR foo" "[{\"dict\": \"Test synonyms\",\"word\":\"test\",\"definition\":\"\\\nresult of test\"}]"
```

Comment:
Should that be `\\\\n` or just `\n`? `\\\n` has a wrong look to it. (Keep in mind that explicitly escaping like `\\n` is better.)

Reply:
It used to be just `\n`, but that didn't work - it got printed as newline leading jq to complain about it being used in the JSON value.

Reply:
Yes, but just `\n` is not a good idea stylistically even though it works. See https://github.com/koalaman/shellcheck/wiki/SC1117

But I typod. I meant to write "Should that be `\\\\n` or just `\\n`?" Awkward. :-P